Routes edits by stu 12.16.24

StuvaScript opened the pull request.

StuvaScript committed:
Initial commit

StuvaScript committed:
Edit some things to work with frontend

Commented out line 83 in modals/User.js file
according to an error in the console stating that
Mongoose.Schema implicitly creates a new email
index based on line 29 stating "unique: true"

Capitalize User.js file in modals folder.

Capitalize ItineraryItem route in itineraryItem.js

StuvaScript committed:
Changed named export to default export

In errors/unauthenticated.js file, changed the
export to a default export since we imported
it as a default import in middleware/
authentication.js

jgoldenberg29 reviewed (approved):
Everything looks good to me Sisi! keep it up! Very nice error handling!

jgoldenberg29 reviewed (changes requested):
looks good to me. The only thing I would change is the error message for incorrect password should say "invalid credentials" like the incorrect email one. We don't really want people to know which piece is incorrect. This might be the same on routes branch and I just missed it. Good work stu 👍

StuvaScript committed:
Edited controllers/itineraryItem.js file

Changed line 19 from 'req.user.userId' to
'req.user' to grab the correct info.

Changed line 26 from 'createdBy: userId' to
'user: userId' to match the model's schema.

StuvaScript committed:
Add code to return user ID

StuvaScript committed:
Changed invalid password error message

StuvaScript committed:
Changed invalid password message

Sisi-tech commented:
> invalid credentials

I corrected the error message. thx

StuvaScript committed:
Merge pull request #33 from Code-the-Dream-School/Itinerary_edits_Stu_12.17.24

Itinerary edits stu 12.17.24

Sisi-tech committed:
Merge branch 'Routes' into Routes_edits_by_Stu_12.16.24
